library/Cube/Web/Controller.php

```diff
@@ -273,7 +272,11 @@ private function transformLegacyDimensionParamsAndRedirect(array $legacyDimensio
 
     public function createTabs(): Tabs
     {
-        $params = Url::fromRequest()->getParams()->toString();
+        $params = Url::fromRequest()
+            ->onlyWith(['dimensions', 'problems', 'sort'])
```

Comment:
Please use `$this->preserveParams`. You're not preserving `showSettings` like the ido controller does.